src/cmd/build.js

```diff
@@ -71,14 +72,14 @@ export async function getDefaultLocalizedName(
   let extensionName: string = manifestData.name;
 
   try {
-    messageContents = await fs.readFile(messageFile);
+    messageContents = await fs.readFile(messageFile, {encoding: 'utf-8'});
   } catch (error) {
     throw new UsageError(
       `Error reading messages.json file at ${messageFile}: ${error}`);
   }
 
   try {
-    messageData = parseJSON(String(messageContents), messageFile);
+    messageData = parseJSON(stripJsonComments(messageContents), messageFile);
```

Comment:
@Rob--W Do you mind to add an additional test case for this fix in 'tests/unit/test-cmd/test.build.js'?